doc/api/worker_threads.md

```diff
@@ -1957,6 +1957,36 @@ this matches its values.
 
 If the worker has stopped, the return value is an empty object.
 
+### `worker.startCpuProfile(name)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* name: {string}
+* Returns: {Promise}
+
+Starting a CPU profile with the given `name`, then return a Promise that fulfills
+with an error or an object which has a `stop` method. Calling the `stop` method will
```

Comment:
Might want to document an actual _object_ for that with its own methods and such. That way if we decide to add more methods or method parameters in the future we would have somewhere to document those additional things. For example, I would like to add support for outputting pprof data in a buffer, or possibly the OTel profiling format (which is heavily based on pprof) in the future if/when that stabilizes at some point.

To that end, and thinking toward the future, it may _also_ make sense for the `stop()` method to return a container object with a `json()` method on it instead, similar to fetch, so other format methods could be added in the future.